socorro/external/postgresql/setupdb_app.py

```diff
@@ -410,6 +346,85 @@ def create_table(element, compiler, **kw):
             db.set_table_owner(self.config.database_username)
             db.set_default_owner(database_name, self.config.database_username)
             db.set_grants(self.config)  # config has user lists
+        return True
```

Comment:
This code was moved from `main`, then I changed the returns so it returns `True` if everything worked. Otherwise it's identical.